packages/core/entity/FileDetail/index.ts

```diff
@@ -116,6 +122,13 @@ export default class FileDetail {
         if (path === undefined) {
             throw new Error("File Path is not defined");
         }
+
+        // AICS FMS files have paths like staging.files.allencell.org/130/b23/bfe/117/2a4/71b/746/002/064/db4/1a/danny_int_test_4.txt
```

Comment:
could add the specific metadata name 'file_path' here

Reply:
Resolved in [8817332](https://github.com/AllenInstitute/biofile-finder/pull/375/commits/88173323832b26611f456ca2a1aaf2638264255f)